modules/proc-switching-bf2-nic.adoc

```diff
@@ -0,0 +1,96 @@
+====
+
+.Prerequisites
+
+* You have installed the SR-IOV Operator. For more information, see "Installing SR-IOV Network Operator". 
+* You have updated Bluefield-2 to the latest firmware. For more information, see link:https://network.nvidia.com/support/firmware/bluefield2/[Firmware for NVIDIA BlueField-2]. 
+
+.Procedure
+
+. Add the following labels to each of your worker nodes by entering the following commands:
++
+[source,terminal]
+----
+$ oc label node <example_node_name_one> node-role.kubernetes.io/sriov=
+----
++
+[source,terminal]
+----
+$ oc label node <example_node_name_two> node-role.kubernetes.io/sriov=
+
+----
+
+. Create a machine config pool for the SR-IOV Operator, for example: 
++
+[source,terminal]
+----
+
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfigPool
+metadata:
+  name: sriov
+spec:
+  machineConfigSelector:
+    matchExpressions:
+      - {key: machineconfiguration.openshift.io/role, operator: In, values: [worker,sriov]}
+  nodeSelector:
+    matchLabels:
+            node-role.kubernetes.io/sriov: ""
+----
+
+. Apply the following `machineconfig.yaml` file to the worker nodes:
++
+[source,yaml]
+----
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfig
+metadata:
+  labels:
+    machineconfiguration.openshift.io/role: sriov
+  name: 99-bf2-dpu
+spec:
+  config:
+    ignition:
+      version: 3.2.0
+    storage:
+      files:
+      - contents:
+          source: data:text/plain;charset=utf-8;base64,ZmluZF9jb250YWluZXIoKSB7CiAgY3JpY3RsIHBzIC1vIGpzb24gfCBqcSAtciAnLmNvbnRhaW5lcnNbXSB8IHNlbGVjdCgubWV0YWRhdGEubmFtZT09InNyaW92LW5ldHdvcmstY29uZmlnLWRhZW1vbiIpIHwgLmlkJwp9CnVudGlsIG91dHB1dD0kKGZpbmRfY29udGFpbmVyKTsgW1sgLW4gIiRvdXRwdXQiIF1dOyBkbwogIGVjaG8gIndhaXRpbmcgZm9yIGNvbnRhaW5lciB0byBjb21lIHVwIgogIHNsZWVwIDE7CmRvbmUKISBzdWRvIGNyaWN0bCBleGVjICRvdXRwdXQgL2JpbmRhdGEvc2NyaXB0cy9iZjItc3dpdGNoLW1vZGUuc2ggIiRAIgo=
+        mode: 0755
+        overwrite: true
+        path: /etc/default/switch_in_sriov_config_daemon.sh
+    systemd:
+      units:
+        - name: dpu-switch.service
+          enabled: true
+          contents: |
+            [Unit]
+            Description=Switch BlueField2 card to NIC/DPU mode
+            RequiresMountsFor=%t/containers
+            Wants=network.target
+            After=network-online.target kubelet.service
+            [Service]
+            SuccessExitStatus=0 120
+            RemainAfterExit=True
+            ExecStart=/bin/bash -c '/etc/default/switch_in_sriov_config_daemon.sh nic || shutdown -r now'
+            Type=oneshot
+            [Install]
+            WantedBy=multi-user.target <1>
+----
+
+<1> (Optional). You can add a PCI address behind `nic`. By default, the first device is selected. If there is more than one device, you must specify which PCI address to be used. The PCI address must be the same on all nodes that switching Bluefield-2 from DPU mode to NIC mode. 
```

Comment:
Change to: "... on all nodes that are switching Bluefield-2 from DPU mode to NIC mode. "